### PR TITLE
Added $lanify

### LINKED ISSUE #30: $zoom(other)
Add the following $zoom commands:
- mar
- arun

### PR DESCRIPTION
**Pull request by AardeYamz**
Added $lanify using image placed in the meme folder, still using tim_scale

### CODE REVIEW

**Comment by coffincw** [comment]
Hi Yannis,

Can you pull from master to your fork so that you have the most up to date code?  And this can be merged easily

**Comment by AardeYamz** [comment]
I believe that I have done that just now.

**Comment by AardeYamz** [comment]
Added additional features and aliases

File: src/bot.py [hunk 1]
```diff
@@ -528,6 +531,15 @@ def get_top_10(data):
...
     await custom_meme.ify(ctx, face_detection.tim_scale, 'memes/tim/tim-face.png', 'timify.png', args)
 
+@client.command(name='lanify', aliases = ['lan-ify'])
+async def lanify(ctx, *args):
+    """Command to paste lan's face on top of faces in an inputed image using facial recognition
+
+       Args:
+        - ctx: context that the command occured use this to access the message and other attributes
+    """
+    await ify(ctx, tim_scale, 'memes/lan/lan-face.png', 'lan-face.png', args)
```
Comment (coffincw) [inline src/bot.py]: Does not work, functions and variables from other files need to be referenced. Should be:
```suggestion
    await custom_meme.ify(ctx, face_detection.lan_scale, 'memes/lan/lan-face.png', 'lan-face.png', args)
```
Also please add this scale underneath `tim_scale` in _face_detection.py_
`lan_scale = (1.55, 0.45, 0.4)`

File: src/bot.py [hunk 2]
```diff
@@ -546,6 +558,24 @@ def get_top_10(data):
     """
     await custom_meme.zoomcam(ctx, 'memes/barrington/zoombarr.png', 'zoombarr_final.png', args)
 
+@client.command(name='zoommarius', aliases=['streamermarius', 'zoom-marr', 'zoom-mar', 'zoom-marius'])
```
Comment (coffincw) [inline src/bot.py]:
```suggestion
@client.command(name='zoommar', aliases=['streamermarius', 'zoommarius', 'zoom-marr', 'zoom-mar', 'zoom-marius'])
```

File: src/bot.py [hunk 3]
```diff
@@ -546,6 +558,24 @@ def get_top_10(data):
...
+async def zoommarius(ctx, *args):
+    """Command to paste an image of marius from Zoom onto the given image in the top right corner
+
+       Args:
+        - ctx: context that the command occured use this to access the message and other attributes
+    """
+    await custom_meme.zoomcam(ctx, 'memes/marius/zoommarius.png', 'zoommarius_final.png', args)
+
+@client.command(name='zoommarun', aliases=['streamerarun', 'zoom-arun', 'zoom-adunna', 'zoomadunna'])
```
Comment (coffincw) [inline src/bot.py]:
```suggestion
@client.command(name='zoomarun', aliases=['streamerarun', 'zoom-arun', 'zoom-adunna', 'zoomadunna'])
```

File: src/bot.py [hunk 4]
```diff
@@ -546,6 +558,24 @@ def get_top_10(data):
...
+async def zoommarius(ctx, *args):
```
Comment (coffincw) [inline src/bot.py]:
```suggestion
async def zoomarun(ctx, *args):
```
duplicate function name

File: src/bot.py [hunk 5]
```diff
@@ -199,8 +199,11 @@
         '*surprisedpikachu [image/url]*': 'The bot uses computer vision through the OpenCV library to put surprised pikachu on identified faces in the inputed image',
         '*marify [image/url]*': 'The bot uses computer vision through the OpenCV library to put marius on identified faces in the inputed image',
         '*timify [image/url]*': 'The bot uses computer vision through the OpenCV library to put tim on identified faces in the inputed image',
+        '*lanify [image/url]*': 'The bot uses computer vision through the OpenCV library to put lan on identified faces in the inputed image',
         '*liamify [image/url]*': 'The bot uses computer vision through the OpenCV library to put liam on identified faces in the inputed image',
         '*zoombarr [image/url]*': 'Pastes an image of barr from Zoom onto the given image in the top right corner',
+        '*zoommarius [image/url]*': 'Pastes an image of marius from Zoom onto the given image in the top right corner',
```
Comment (coffincw) [inline src/bot.py]:
```suggestion
        '*zoommar [image/url]*': 'Pastes an image of marius from Zoom onto the given image in the top right corner',
```

**Review by coffincw — CHANGES_REQUESTED** [review]
Looks good, just need to make a few adjustments.

**Comment by AardeYamz** [comment]
Made suggested modifications

**Review by coffincw — APPROVED** [review]
Looks good!

**Pull request merged**